Lombiq.Tests.UI/Extensions/ShortcutsUITestContextExtensions.cs

```diff
@@ -117,22 +117,23 @@ public static async Task<string> GetCurrentUserNameAsync(this UITestContext cont
     public static Task SetUserRegistrationTypeAsync(
         this UITestContext context,
         UserRegistrationType type,
-        string tenant = "Default",
+        string tenant = null,
         bool activateShell = true) =>
-            context.Application.UsingScopeAsync(
-                async serviceProvider =>
-                {
-                    var siteService = serviceProvider.GetRequiredService<ISiteService>();
-                    var settings = await siteService.LoadSiteSettingsAsync();
+        UsingScopeAsync(
+            context,
+            async serviceProvider =>
+            {
+                var siteService = serviceProvider.GetRequiredService<ISiteService>();
+                var settings = await siteService.LoadSiteSettingsAsync();
 
-                    settings.Alter<RegistrationSettings>(
-                        nameof(RegistrationSettings),
-                        registrationSettings => registrationSettings.UsersCanRegister = type);
+                settings.Alter<RegistrationSettings>(
+                    nameof(RegistrationSettings),
+                    registrationSettings => registrationSettings.UsersCanRegister = type);
 
-                    await siteService.UpdateSiteSettingsAsync(settings);
-                },
-                tenant,
-                activateShell);
+                await siteService.UpdateSiteSettingsAsync(settings);
+            },
+            tenant,
+            activateShell);
```

Comment:
In all of these blocks, the only change is using `UsingScopeAsync()` and removing the unnecessary indentation.